lib/integration/scenarios/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -1624,13 +1583,13 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
                     ( \case
                         [acc] -> do
                             (acc ^. #_index) `shouldBe` 0
-                            (acc ^. #_stake) `shouldBe` ApiAmount 0
+                            (acc ^. #_stake) `shouldBe` balance
```

Comment:
this is because now we are having base addr (ie. payment cred + stake cred) and due to that stake key have ALL funds on itself, so what fixture has, so balance. Correct? And previously we used enteprise addr, ie. those that use payment cred only, so stake addr assume 0. Correct?